zircon-object/src/signal/futex.rs

```diff
@@ -419,25 +419,4 @@ mod tests {
         assert!(Arc::ptr_eq(&futex.owner().unwrap(), &thread));
         assert_eq!(futex.wake(1), 0);
     }
-
-    #[async_std::test]
-    async fn time_out() {
```

Comment:
Can we add this test back?

Reply:
I decided to remove this test given that timeout is a feature in `blocking_run`, and it has been well tested there.

The lack of this test will not have much impact on the test coverage (only -0.88%).

Reply:
OK